src/cmds/init.py

```diff
@@ -155,18 +166,6 @@ def migrate_from(
         return 0
     print(f"\n{old_root} found")
     print(f"Copying files from {old_root} to {new_root}\n")
-    copied = []
```

Comment:
We use this generically to copy e.g. the databases over so I think you can't remove it. What we do is just change the URIs as needed down below

Reply:
Maybe I'm missing something but I think this change would keep db/blockchain_v23.db from copying over?

Reply:
Tested my refactored code, it copies the db just fine.
The `copied = []` was legacy variable that was eliminated when I refactored the code. I did entire search for this variable and it isn't used anywhere. Here's the original use: https://github.com/Chia-Network/chia-blockchain/commit/e2b3b650c6ebd78a041c36b322c49f04e1d8b5d3